core/templates/dev/head/domain/question/QuestionObjectFactorySpec.ts

```diff
@@ -18,22 +18,25 @@
 
 // TODO(YashJipkate) Remove the following block of unnnecessary imports once
 // QuestionObjectFactory.ts is upgraded to Angular 8.
+import { MisconceptionObjectFactory } from
+  'domain/skill/MisconceptionObjectFactory.ts';
 import { RuleObjectFactory } from 'domain/exploration/RuleObjectFactory.ts';
 import { WrittenTranslationObjectFactory } from
   'domain/exploration/WrittenTranslationObjectFactory.ts';
 // ^^^ This block is to be removed.
 
 require('domain/question/QuestionObjectFactory.ts');
-require('domain/skill/MisconceptionObjectFactory.ts');
 
 describe('Question object factory', function() {
   var QuestionObjectFactory = null;
   var _sampleQuestion = null;
   var _sampleQuestionBackendDict = null;
-  var MisconceptionObjectFactory = null;
+  var misconceptionObjectFactory = null;
```

Comment:
These are the AngularJS specs and do not have the types associated with them elsewhere. Typing these will introduce an inconsistency. Also, since these are to be replaced with their Angular equivalents, I think we should let the AngularJS part untyped for consistency.